adapter/implementations/Terminator.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from adapter.base import TerminalAdapterInterface
+
+class Terminator(TerminalAdapterInterface):
+    filename = "$HOME/.config/terminator/config"
+    setting_field = "background_image"
+
+    @staticmethod
+    def is_available():
+        return "TERMINATOR_UUID" in os.environ
+
+    def set_pokemon(self, pokemon):
```

Comment:
The [adapter pattern has changed](https://github.com/LazoCoder/Pokemon-Terminal/blob/master/scripter.py#L39) to accept an image_file_path instead of a pokemon.  See #83